src/main/bugfree/Bugfree.php

```diff
@@ -45,7 +45,7 @@ public function __construct(Resolver $resolver, Config $config)
     public function parse($name, $source)
     {
         $result = new Result($name, $this->config);
-        $parser = new Parser(new Lexer());
+        $parser = (new ParserFactory())->create(ParserFactory::ONLY_PHP7, new Lexer());
```

Comment:
Someone might want to use php7 to run bugfree on a codebase that still supports 5. I'll just change it to PREFER_PHP7.